sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -672,24 +679,24 @@ bool _pi_queue::isInOrderQueue() const {
           0);
 }
 
-pi_result _pi_queue::resetCommandListFenceEntry(
-    _pi_queue::command_list_fence_map_t::value_type &MapEntry,
-    bool MakeAvailable) {
-  bool UseCopyEngine = MapEntry.second.IsCopyCommandList;
+pi_result _pi_queue::resetCommandList(pi_command_list_ptr_t CommandList,
```

Comment:
Same comment as above. Please rename if you think that's ok.

Reply:
I'd like to stay with short name. CommandList is basically 2 things, the L0 handle and PI info. There is no "map" or "iterator" properties that we use in it.